lib/base-compiler.ts

```diff
@@ -826,6 +827,10 @@ export class BaseCompiler {
         );
     }
 
+    fixIncompatbleOptions(options: string[], userOptions: string[]) {
```

Comment:
Just to keep things as nice and encapsulated as possible I would like for this to be `protected`

Reply:
agreed